addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/ahamodel/HeatingModel.java

```diff
@@ -103,7 +103,7 @@ public String getRadiatorMode() {
         } else if (TEMP_FRITZ_MAX.compareTo(tsoll) == 0) {
             return MODE_BOOST;
         } else {
-            return MODE_UNKNOWN;
+            return MODE_ON;
```

Comment:
What falls in this category?

Reply:
The first four cases handle fixed temperature values (e.g. defined eco or comfort temperatures). Hence we have expected modes which should be returned. The last else statement handles all temperatures between the min and max temperature, which doesn't belong to the first four cases. But the mode should not have the value `unknown,` because we know that the radiator is `on`.